BDArmory/UI/BDArmorySetup.cs

```diff
@@ -3467,8 +3467,44 @@ void WindowSettings(int windowID)
                 {
                     CheatOptions.InfiniteElectricity = BDArmorySettings.INFINITE_EC;
                 }
+                if (!BDArmorySettings.G_LIMITS && (HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().GPartLimits || HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().GKerbalLimits))
+                    BDArmorySettings.G_LIMITS = true; 
+                BDArmorySettings.PART_GLIMIT = HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().GPartLimits; // Sync with the Game Difficulty window if the checkbox was toggled there.
+                BDArmorySettings.KERB_GLIMIT = HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().GKerbalLimits;
+                BDArmorySettings.G_TOLERANCE = HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().KerbalGToleranceMult * 20.5f;
+                if (BDArmorySettings.G_LIMITS != (BDArmorySettings.G_LIMITS = GUI.Toggle(SLeftRect(line), BDArmorySettings.G_LIMITS, StringUtils.Localize("#LOC_BDArmory_Settings_GLimitsMode"))))//G-Force Limits
+                {
+                    if (!BDArmorySettings.G_LIMITS)
+                    {
+                        HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().GPartLimits = false;
+                        HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().GKerbalLimits = false;
+                        HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().KerbalGToleranceMult = 1f;
+                        BDArmorySettings.KERB_GLIMIT = false;
+                        BDArmorySettings.PART_GLIMIT = false;
+                    }
+                }
+                if (BDArmorySettings.G_LIMITS)
+                {
+                    if (BDArmorySettings.PART_GLIMIT != (BDArmorySettings.PART_GLIMIT = GUI.Toggle(SLeftRect(++line, 1), BDArmorySettings.PART_GLIMIT, StringUtils.Localize("#autoLOC_140950"))))//Part G-Force Limits
+                    {
+                        HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().GPartLimits = BDArmorySettings.PART_GLIMIT;
+                    }
+                    if (BDArmorySettings.KERB_GLIMIT != (BDArmorySettings.KERB_GLIMIT = GUI.Toggle(SRightRect(line, 1), BDArmorySettings.KERB_GLIMIT, StringUtils.Localize("#autoLOC_140953"))))//Kerbal G-Force Limits
+                    {
+                        HighLogic.CurrentGame.Parameters.CustomParams<GameParameters.AdvancedParams>().GKerbalLimits = BDArmorySettings.KERB_GLIMIT;
+                    }
+                    if (BDArmorySettings.KERB_GLIMIT)
+                    {
+                        GUI.Label(SLeftSliderRect(++line, 1), $"{StringUtils.Localize("#autoLOC_140956")}:  ({BDArmorySettings.G_TOLERANCE})", leftLabel);//Kerbal G-Force Tolerance
```

Comment:
This could use a `({BDArmorySettings.G_TOLERANCE:0.0}g)` format specifier and unit.